src/amo/pages/UserProfileEdit/index.js

```diff
@@ -207,13 +206,15 @@ export class UserProfileEditBase extends React.Component<Props, State> {
           toPath = `/${toPath}`;
         }
         try {
-          history.push(toPath);
+          this.props._window.location.assign(toPath);
           return;
         } catch (error) {
           log.warn(`Error redirecting to location: ${toPath}: ${error}`);
```

Comment:
Is this still possible?

Reply:
It can in theory (https://developer.mozilla.org/en-US/docs/Web/API/Location/assign), and I thought I saw it once when testing with a `to` of `/en-US/developers/` locally, but now I cannot reproduce that.

Because `location.assign()` can throw an exception, it's probably safest to keep the `try/catch` imo.

Reply:
I agree. Shouldn't we add a try/catch block around the other call then?

Reply:
We could. I didn't because we are in control of that URL - it's not based on a `to` URL parameter. Therefore there's no reason to expect it will *ever* fail. In fact, the way this code is written, that second call is what we do if the first call fails. So I guess we could add an additional `try/catch` around it, but I'm not sure what we'd do in that catch other than log another warning. Do you think that's worth doing?

Reply:
gotcha, let's keep things as is then.